samples/previews/azure-functions/service-bus/ASBFunctions_0/AzureFunctions.ASBTrigger.Static/CustomComponent.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+public class CustomComponent
+{
+    readonly string initialValue;
+
+    public CustomComponent(string initialValue)
+    {
+        this.initialValue = $"{initialValue} {DateTime.UtcNow:O}";
```

Comment:
what's the purpose of the timestamp?

Reply:
This is to demonstrate that the Singleton scope is actually working 🙂